allele_counts_from_strelka.py

```diff
@@ -14,131 +15,48 @@ def main():
         "-v", dest="vcfFile", action="store", type="string", help="path to strelka vcf file."
     )
 
-    # get at the arguments
+    # Get at the arguments
     options, _ = parser.parse_args()
     vcfFile = options.vcfFile
-    if vcfFile is None:
+    if not vcfFile:
         parser.print_help()
         sys.exit()
 
-    # Check that our vcf file exists
+    # Check that our bam file exists
     if not os.path.isfile(vcfFile):
         sys.stderr.write(vcfFile + " is not a valid file\n")
         sys.exit()
 
     # Loops over the lines and pull out the called allele counts
-    basename = os.path.basename(vcfFile)
-    af_0p0_0p05_file = open("%s_%s.vcf" % (basename, "af_0p0_0p05"), "w")
```

Comment:
This script used to create a sequence of files with accumulated counts.  The new version seems to create a single file report.   The original file is the one intended to be used in the TMBur output.

Reply:
Ok @RichardCorbett I've reverted the file.